test/test-PlacesProvider.js

```diff
@@ -100,6 +100,53 @@ exports.test_Links_getTopFrecentSites_Order = function*(assert) {
   }
 };
 
+exports.test_Links_getRecentLinks = function*(assert) {
+  let provider = PlacesProvider.links;
+  let {
+    TRANSITION_TYPED,
+    TRANSITION_LINK
+  } = PlacesUtils.history;
+
+  function timeDaysAgo(numDays) {
+    let now = new Date();
+    return now.getTime() - (numDays * 24 * 60 * 60 * 1000);
```

Comment:
Since we're now doing this in more than 1x place, not sure if we want to make this magic math into a const like `const DAYS_IN_MS = 24 * 60 * 60 * 1000;` and then convert this to:

```js
return now.getTime() - (numDays * DAYS_IN_MS);
```